### PR TITLE
document-download-frontend docker image

### PR DESCRIPTION
**Pull request by playfair0**
- additional context: https://trello.com/c/WGCSkzLt/121-create-dockerfile-for-document-download-frontend

---

🚨⚠️ This will be deployed automatically all the way to production when you click merge ⚠️🚨

For more information, including how to check this deployment on preview or staging first before it goes to production, see our [team wiki section on deployment](https://github.com/alphagov/notifications-manuals/wiki/Merging-and-deploying#deployment)

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+RUN echo "Installing python dependencies" \
+    && pip install -r requirements.txt
+
+COPY app app
+
+COPY application.py gunicorn_config.py ./
+COPY --from=frontend-build /usr/frontend/app/static app/static
+COPY --from=frontend-build /usr/frontend/app/templates app/templates
+
+USER notify
```
Comment (samuelhwilliams) [inline docker/Dockerfile:38]: 💅 be nice to keep a blank line at the end here

**Review by samuelhwilliams — COMMENTED** [review]
Dockerfile looks good 👍 nice use of multiple stages to get the static assets without the node modules.

Do we also need a run-with-docker script and makefile step?

**Review by BlessedDev — APPROVED** [review]

**Comment by samuelhwilliams** [comment]
Do we want to also start adding a standard `.dockerignore` file to our repos to exclude things like `.git`, `node_modules`, `__pycache__`, `*.pyc` etc?

**Comment by BlessedDev** [comment]
> Do we want to also start adding a standard `.dockerignore` file to our repos to exclude things like `.git`, `node_modules`, `__pycache__`, `*.pyc` etc?
> 
> Maybe see:
> 
> * https://github.com/GoogleCloudPlatform/getting-started-python/blob/main/optional-kubernetes-engine/.dockerignore
> * https://gist.github.com/allanchua101/0ff808ea0aa3d6db7fd5a6b6e004fec1

It is a good suggestion although it is outside the scope of this ticket. We could open up a new ticket to address it as there are advantages to it.

**Comment by samuelhwilliams** [comment]
I don't think it needs to be out of scope - having a new ticket probably introduces more overhead than just putting a standard dockerignore in place here. But one for you/your team to manage 👍

**Pull request merged**